lib/ui/painting/codec.cc

```diff
@@ -466,6 +499,25 @@ Dart_Handle MultiFrameCodec::getNextFrame(Dart_Handle callback_handle) {
   return Dart_Null();
 }
 
+void MultiFrameCodec::clearAndDisableFrameCache() {
```

Comment:
clearAndDisableFrameCache and enableFrameCache are running on the UI thread, while the IO thread might be accessing frameBitmaps_.
I believe we need some sort of synchronization here (or to post these 2 to the IO thread)

Reply:
That would have not been a good one to debug, thanks. Posting these on the IO thread instead now.